src/Engine/MapController.java

```diff
@@ -195,11 +198,19 @@ private void handleMapInput(InputHandler.InputAction input)
             nextSelectedUnitIndex = 0;
           }
 
-          // If we find a unit that is ready to go, move the cursor to it.
-          Unit nextUnit = myGame.activeCO.units.get(nextSelectedUnitIndex);
-          if( !nextUnit.isTurnOver )
+          if( nextSelectedUnitIndex < numUnits )
```

Comment:
`nextSelectedUnitIndex` isn't really correct anymore... But I'm not sure what a better option would be. Maybe `nextSeekLocation`?